### ISSUE TITLE
Create logos for use on light and dark backgrounds

### ISSUE THREAD

**Issue by jungshadow** [it-dept-cis/Essential-Guide-to-Election-Security#38]
The new theme features switchable light and dark modes. However, the current logo—which may not be the final logo we use—doesn't have a high contrast ratio on the dark background (see appended images below). Currently, the same logo file is used in the configuration settings:

```py
html_theme_options = {
    'light_logo': 'cis-logo.png',
    'dark_logo': 'cis-logo.png',
}
```

We need to decide which logo will inherit that space and there needs to be versions of the logo for both light (white) and dark (black) backgrounds.

![image](https://user-images.githubusercontent.com/85972/161863675-4d6b057a-7535-438f-91b9-9c120471f220.png)

![image](https://user-images.githubusercontent.com/85972/161863689-bcf31c43-c152-4a66-b52c-ae147f1a5e57.png)

**Comment by mike-garcia** [comment]
@jungshadow see _static folder

**Comment by jungshadow** [comment]
This is fixed via #41
